tests/test_history.py

```diff
@@ -812,6 +812,23 @@ def test_minute_dividends(self):
         # last 5 minutes should not be adjusted
         np.testing.assert_array_equal(np.array(range(782, 787)), window3[395:])
 
+    def test_overnight_adjustments(self):
+
+        # Should incorporate adjustments on midnight 01/06
+        last_traded = pd.Timestamp("2015-01-05 21:00", tz='UTC')
```

Comment:
Can we add a test asserting that this correctly handles the window boundary crossing a non-trading day?